src/graphql/query/thanksPage.graphql

```diff
@@ -132,6 +132,14 @@ query checkoutReceipt($checkoutId: Int!, $visitorId: String) {
 			key
 			value
 		}
+		ftd_message_enable: uiConfigSetting(key: "thanks_page_ftd_message_enable") {
```

Comment:
Is there a reason why were using a config instead of an experiment?

Reply:
This is not a test, but we want the ability to turn this off